ChangeLog.md

```diff
@@ -2,6 +2,9 @@
 * v9.51.1 (2015-04-??)
 	* Rename `server.api.log` to `server.access.log`
 	* Add `requests_timeout` parameter
+	* Add `rpc_batch_size` parameter and optimize RPC calls for block parsing and UTXO search
+	* Add `check_asset_conservation` parameter and optimize assets conservation checking
+	* Add `parse_mempool` parameter and disable mempool parsing by default
```

Comment:
Can we just call this param `mempool`? `parse_mempool` sounds like it has something to do with *parsing*.